### ISSUE TITLE
Error when merging account

### ISSUE THREAD

**Issue by clone1018** [liberapay/liberapay.com#774]
URL: https://liberapay.com/migrate?step=2
Email: luke@axxim.net

"Looks like you've found a bug! Sorry for the inconvenience, we'll get it fixed ASAP!"

The details of this error have been recorded. If you decide to contact us please include the following error identification code in your message: e7d8bf1916244f94bec25d495f2e9578.

I started migrating my Gratipay account, I had to click a SSO link in my email and then I came back to the migrate process to finish.

**Comment by Changaco** [comment]
https://sentry.io/share/issue/34c2e876d7ff407687f650ec3058457f/

```
BadUserId: (u'814589697695555584', u'user_id')
  File "site-packages/algorithm.py", line 321, in loop
    new_state = function(**deps.as_kwargs)
  File "aspen/request_processor/algorithm.py", line 73, in render_resource
    return {'output': resource.render(state)}
  File "aspen/http/resource.py", line 78, in render
    return super(Dynamic, self).render(available[0], state)
  File "aspen/simplates/simplate.py", line 121, in render
    exec(self.pages[1], context)
  File "/opt/python/bundle/12/app/www/migrate.spt", line 151, in <module>
    e = platform.get_user_info('', 'user_id', e['user_id'])
  File "liberapay/elsewhere/_base.py", line 280, in get_user_info
    response = self.api_get(domain, path, sess=sess, error_handler=error_handler)
  File "liberapay/elsewhere/_base.py", line 147, in api_get
    return self.api_request('GET', domain, path, sess=sess, **kw)
  File "liberapay/elsewhere/_base.py", line 135, in api_request
    error_handler(response, is_user_session, domain)
  File "liberapay/elsewhere/_base.py", line 278, in error_handler
    raise BadUserId(value, key)
```

Looks like an obsolete Twitter user ID in the data imported from Gratipay.

**Comment by Changaco** [comment]
@clone1018 I've deployed a patch, try again?

**Comment by clone1018** [comment]
It works! Thanks

> The data import from Gratipay has been successfully completed.

**Comment by mattbk** [comment]
Nice work @Changaco. I like all the "recreated as a pledge to x" entries.

**Comment by Changaco** [comment]
For the record this was fixed in bfb2958716669db11a55e8f34aa72388d9a928d2.
